src/planner/operator/logical_distinct.cpp

```diff
@@ -15,6 +15,9 @@ string LogicalDistinct::ParamsToString() const {
 }
 void LogicalDistinct::Serialize(FieldWriter &writer) const {
 	writer.WriteSerializableList(distinct_targets);
+	if (order_by) {
+		throw NotImplementedException("Serializing ORDER BY not yet supported");
```

Comment:
One thing I noticed looking at serializing sorted aggregates is that we are not set up to handle aggregate wrappers. But we may be able to deal with that now?